### PR TITLE
Cosmos Stub Implementation

### PR DESCRIPTION
**Pull request by timolegros**
## Link to Issue
Closes: #5744

## Description of Changes
- Implements the stubs created in #5751.
- Fixed `cosmos-chain-testing/` so that v1beta chain is on port 5050 and v1 chain is on port 5051.
- Added devnet tests for the new token balance fetching methods

## Test Plan
Not manually testable until the class is used in the gating controllers.

## Deployment Plan

1. 

## Other Considerations

-

### CODE REVIEW

**Comment by timolegros** [comment]
This PR is technically ready but I am leaving it in draft until #5751 is merged and a few points are clarified with Ryan.

File: knowledge_base/Package-Scripts.md [hunk 1]
```diff
@@ -67,6 +67,10 @@ If you add a script to the package.json, you must add documentation here, descri
   - [start-all](#start-all)
   - [start-consumer](#start-consumer)
   - [start-evm-ce](#start-evm-ce)
+- [Devnets](#devnets)
+  - [cosmos:build](#comsos:build)
+  - [cosmos:start](#comsos:start)
+  - [cosmos:stop](#cosmos:stop)
```
Comment (mhagel) [inline knowledge_base/Package-Scripts.md:73]:
```suggestion
  - [cosmos:build](#cosmos:build)
  - [cosmos:start](#cosmos:start)
  - [cosmos:stop](#cosmos:stop)
```

File: packages/commonwealth/test/devnet/cosmos/tokenBalanceFetching.spec.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import { use as chaiUse, expect } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { RedisCache } from 'common-common/src/redisCache';
+import models from '../../../server/database';
+import { BalanceSourceType } from '../../../server/util/requirementsModule/requirementsTypes';
+import { TokenBalanceCache } from '../../../server/util/tokenBalanceCache/tokenBalanceCache';
+
+chaiUse(chaiAsPromised);
+
+describe.only('Token Balance Cache Cosmos Tests', () => {
```
Comment (mhagel) [inline packages/commonwealth/test/devnet/cosmos/tokenBalanceFetching.spec.ts:10]:
```suggestion
describe('Token Balance Cache Cosmos Tests', () => {
```

File: packages/commonwealth/test/devnet/cosmos/tokenBalanceFetching.spec.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  const cosmosChainId = 'csdkv1ci';
+  const addressOneBalance = '50000000000';
+  const addressTwoBalance = '30000000000';
+
+  before(async () => {
+    const redisCache = new RedisCache();
+    tbc = new TokenBalanceCache(models, redisCache);
+  });
+
+  // it('should fail if a bech32 prefix is not provided');
```
Comment (mhagel) [inline packages/commonwealth/test/devnet/cosmos/tokenBalanceFetching.spec.ts:25]: remove?

**Review by jnaviask — APPROVED** [review]

**Review by mhagel — COMMENTED** [review]
a couple nits

**Review by CowMuon — APPROVED** [review]

**Review by mhagel — APPROVED** [review]

**Pull request merged**